panda/src/audiotraits/fmodAudioSound.cxx

```diff
@@ -538,6 +540,11 @@ get_name() const {
   return _file_name;
 }
 
+const bool& FmodAudioSound::
+get_positional() const {
```

Comment:
The return type doesn't match the base class method it's overriding; this is returning `const bool &`, but the base class method returns `const bool`.  But I think just returning `bool` would be best.